cmd/earthly/main.go

```diff
@@ -205,7 +205,12 @@ func main() {
 
 	// Load .env into current global env's. This is mainly for applying Earthly settings.
 	// Separate call is made for build args and secrets.
-	if fileutil.FileExists(dotEnvPath) {
+	dotPathExists, err := fileutil.FileExists(dotEnvPath)
+	if err != nil {
+		fmt.Printf("Error checking for file %s: %s\n", dotEnvPath, err.Error())
```

Comment:
If we don't have permission to read `.env` should we print out a warning and continue?

Reply:
Treating it as a warning feels like we would mask a problem; I would rather not make an assumption on the user's behalf and error out. This feels more explicit and less error-prone.

Reply:
Yeah, I think I agree :+1: